pmlc/conversion/stdx_to_llvm/tests/reshape.mlir

```diff
@@ -0,0 +1,85 @@
+// RUN: pmlc-opt -convert-stdx-to-llvm -canonicalize -cse %s | FileCheck %s
+
+module {
+  func @reshaper0(%arg0: memref<1x1x60xf32>) -> memref<60xf32> {
+    %0 = stdx.reshape(%arg0) : (memref<1x1x60xf32>) -> memref<60xf32>
+    return %0 : memref<60xf32>
+  }
+}
+
+// CHECK-LABEL: func @reshaper0
+// CHECK: undef : !llvm<"{ float*, float*, i64, [3 x i64], [3 x i64] }">
+// CHECK: [[index0:%[0-9]+]] = llvm.mlir.constant(60 : index)
```

Comment:
Usually we do `%[[index0:.*]]` because we have no guarantee that the printer will continue to use numbers, but it does always prefix with `%`